packages/playground/src/utils/mintings.ts

```diff
@@ -0,0 +1,87 @@
+export interface receiptInterface {
+  type: string;
+  node_type?: string;
+  node_id?: number;
+  farm_name?: string;
+  farm_id?: number;
+  measured_uptime?: number;
+  period?: {
+    start: number;
+    end: number;
+  };
+  cloud_units?: {
+    cu: number;
+    su: number;
+    nu: number;
+  };
+  resource_units?: {
+    cru: number;
+    mru: number;
+    hru: number;
+    sru: number;
+  };
+  stellar_payout_address?: string;
+  tft_connection_price?: number;
+  minted_cloud_units?: {
+    cu: number;
+    su: number;
+    nu: number;
+  };
+  correct_cloud_units?: {
+    cu: number;
+    su: number;
+    nu: number;
+  };
+  fixup_cloud_units?: {
+    cu: number;
+    su: number;
+    nu: number;
```

Comment:
cloud units which are `cu`, `su`, `nu` are repeated 4 times. why not extract them in an interface and reuse it?